src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -117,7 +120,18 @@ std::string TriggeredSubscription::toString(const std::string& delimiter)
   std::stringstream ss;
 
   ss << throttling << delimiter << lastNotification << delimiter << formatToString(format) << delimiter << reference;  
-  ss << expression.q << delimiter << expression.georel << delimiter << expression.coords << delimiter << expression.geometry << delimiter;
+  ss << expression.georel << delimiter << expression.coords << delimiter << expression.geometry << delimiter;
 
   return ss.str();
 }
+
+
+
+/* ****************************************************************************
+*
+* TriggeredSubscription::stringFilterSet - 
+*/
+void TriggeredSubscription::stringFilterSet(StringFilter* stringFilterP)
+{
+  stringFilter = *stringFilterP;
```

Comment:
I'd suggest a comment above this line:

```
// This is an object copy, like memcpy(&stringFilter, stringFilterP), but including the vectors inside stringFilterP.
```

Probably for C++ people more skilled than me is obvious, but... I think it doesn't hurt.